code/modules/recycling/disposal-unit.dm

```diff
@@ -527,6 +527,179 @@
 		M.forceMove(src)
 	flush()
 
+//Trap Door
+/obj/machinery/disposal/trapdoor
+	name = "trapdoor"
+	desc = "Almost like a door, but on floor."
+	density = 0
```

Comment:
and while you're at it use FALSE instead of 0 since those vars can be either 0 or 1